test/blocks/index.js

```diff
@@ -0,0 +1,73 @@
+import mocha from 'mocha';
+import chai, { expect } from 'chai';
+import dirtyChai from 'dirty-chai';
+
+import * as blocks from '@wordpress/blocks';
+
+chai.use( dirtyChai );
```

Comment:
If we're going to pull in and assume plugins to be used consistently, could we somehow apply it as part of the overall test initialization?